### DISCUSSION TITLE
Support to mobile development

### DISCUSSION THREAD

**Discussion by fregayeg** [reduxjs/redux-toolkit#2505] [Ideas]
Hello! I'm a mobile app developer (React Native), currently, I'm using RTK and its feature Query. I read RTK docs almost everyday, really enjoying it 😍. 

However, I wish I could see some intros, guides or conclusions about mobile development using modern redux, I mean on the docs website, if its possible 🙏. 

Sometimes I find it difficult to follow an example (codesandbox example) from docs, which is ofc a web app, to make such in my mobile environment . I think it could be more better for people like me to find some ready examples in docs for mobile purpose. This [one](https://reactnative.dev/docs/0.64/getting-started#interactive-examples) is a demo with possibility to interact and see result in time. ([Expo snack](https://snack.expo.dev/))

As a **starting point**, I'd suggest to add `mobile` keyword to [this paragraph](https://redux-toolkit.js.org/rtk-query/overview#:~:text=It%20is%20designed%20to%20simplify%20common%20cases%20for%20loading%20data%20in%20a%20web%20application):
> It is designed to simplify common cases for loading data in a web application

So it becomes:
> It is designed to simplify common cases for loading data in a web & **mobile** apps.

Note: I know that many use cases are same as on web, so there are no big differences 🙂. I aim to **what is different**  like [mobile navigation](https://reactnavigation.org/), [notifications](https://github.com/wix/react-native-notifications), [bottom-sheet](https://gorhom.github.io/react-native-bottom-sheet/), [testing ](https://callstack.github.io/react-native-testing-library/) and for me the prior topic, could be about **fetching & caching data**. 

The advantages:
- ٌHelp any developers to discover what is common between web and mobile and what is not.
- Help mobile developers to use redux with less pain. 
- Redux will appear more powerful and effective compared to other tools.
- Attract the attention of mobile developers around the world, especially those working on large and complex systems.
- Get more contributors and fans 🤩

This idea is open to any criticism, enrichment, feedback

**Comment by markerikson** [comment]
I'd _love_ to have some RN example projects and docs pages.  

Problem is, I'm the only one actively doing any work on our docs, and I've got a ton of other things to do.  For that matter, I have no experience working with RN myself.

So, this really needs to be something that the community contributes.

**Reply by fregayeg** [reply]
Sounds Great! I just wanted to know if its possible.. Now I think I'm going to try something soon **by myself**, then will submit a PR. This discussion is still open to anyone who wants to participate.
Thank you for your answer @markerikson
